ci-operator/step-registry/nutanix/validate-credential-rotation/nutanix-validate-credential-rotation-commands.sh

```
@@ -0,0 +1,380 @@
+# Load existing cluster credentials and context
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+source ${SHARED_DIR}/nutanix_context.sh
+
+# Disable tracing for credential operations
+set +x
+source /tmp/nutanix-creds/secrets.sh
+
+# Set up new credentials (either from env vars or simulate with existing)
+if [[ -z "${NEW_NUTANIX_USERNAME:-}" ]]; then
+  echo "No NEW_NUTANIX_USERNAME provided, simulating rotation with existing credentials"
+  NEW_NUTANIX_USERNAME="${prism_central_username}"
```

Shellcheck (0.11.0)

[warning] prism_central_username is referenced but not assigned.

(SC2154)